docs/terms/Public-Engagement.md

```diff
@@ -14,6 +14,9 @@ Focus group
 Informed consent
     A process by which a participant voluntarily confirms his or her willingness to participate in a particular trial, after having been informed of all aspects of the trial that are relevant to the decision to participate {cite}`NIHR2022`. 
 
+Plain English Summary
+        A plain English summary is a clear, brief summary of the research that has been written for members of the public, rather than researchers or professionals. It should be written clearly and simply, without jargon and with an explanation of any technical terms {cite}NIHR2022. This is sometimes also known as a "Plain Language Summary" or "Lay Summary", however the term "Lay Summary" is no longer used by most communities.  
```

Comment:
Is the indentation too much? It doesn't match the others, but I don't know if that matters?

Reply:
Very minor change, I would change first sentence to this (add the word 'for')

A plain English summary is a clear, brief summary of the research that has been written for members of the public, rather than **_for_** researchers or professionals.